.gitignore

```diff
@@ -8,6 +8,7 @@ npm-debug.log
 *.egg-info/
 .venvs/
 dist/
+build-with-tests/
```

Comment:
Leftover?

Reply:
Nope. Let it be. Added build rules to package.json